sklearn/metrics/_base.py

```diff
@@ -246,6 +246,6 @@ def _check_pos_label_consistency(pos_label, y_true):
             "{-1, 1} or pass pos_label explicitly."
         )
     elif pos_label is None:
-        pos_label = 1.0
+        pos_label = 1
```

Comment:
Was this a bug?

Reply:
It would not show a floating label on the plot. The code itself would probably make some casting but was running as expected.